Docs: document editing cortex/loki recording rules, namespaces and groups (#38547)
grafana/grafana: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md

+++
title = "Edit Cortex or Loki rule groups and namespaces"
description = "Edit Cortex or Loki rule groups and namespaces"
keywords = ["grafana", "alerting", "guide", "group", "namespace", "cortex", "loki"]
weight = 400
+++

# Edit Cortex or Loki rule groups and namespaces

Cortex or Loki rule groups and namespaces can be edited via the Grafana alerting UI. Currently renaming namespaces and groups and editing group evaluation interval is supported.

## Rename a namespace

1. In the Grafana menu hover your cursor over the Alerting (bell) icon.

> 1. Find any group form the namespace you want to edit and click the edit (pen) icon.

Reviewer: How do you find the namespace?

> 1. Find any group form the namespace you want to edit and click the edit (pen) icon.

Author: So groups are children of namespaces, that is, you can have several namespaces and each namespace can contain several groups. But in the UI we only list groups, named "namespace / group". So to edit namespace name you have to find a group that belongs to that namespace, click "edit" and namespace name input shows up along with group inputs.

Does that make sense? How do we explain it better in docs?

> 1. Find any group form the namespace you want to edit and click the edit (pen) icon.

Reviewer: Added an introduction to the section based on your explanation here.

> 1. Enter new name in the  **Namespace** input and click **Save changes**.

Reviewer: ```suggestion
1. Enter a new name in the  **Namespace** field, then click **Save changes**.
```

> A new namespace will be created, all groups will be copied into it from the old namespace and the old namespace will be deleted.

Reviewer: ```suggestion
A new namespace is created and all groups are copied into this namespace from the old one. The old namespace is deleted.
```

## Rename rule group or change rule group evaluation interval

> Rules within a group are run sequentially at a regular interval, by default every 1 minute. This interval can be changed in the UI.

Reviewer: ```suggestion
The rules within a group are run sequentially at a regular interval, the default interval is one (1) minute. You can modify this interval using the following instructions.
```

1. In the Grafana menu hover your cursor over the Alerting (bell) icon.
1. Find the group you want to edit and click the edit (pen) icon.

> 1. Modify **Rule group** and **Rule group evaluation interval** inputs as necessary and click **Save changes**.

Reviewer: ```suggestion
1. Modify the **Rule group** and **Rule group evaluation interval** information as necessary.
1. Click **Save changes**.
```

> If group was renamed, a new group will be created containing all the rules from the old one, and the old group deleted.

Reviewer: ```suggestion
If you remaned the group, a new group is created that has all the rules from the old group, and the old group deleted.
```

> 

Reviewer: Can we have some text separating the two images?

Author: I'm not sure what it could be :see_no_evil:  suggestions?

Reviewer: We can probably leave this and update it after.

+++
title = "Edit Cortex or Loki rule groups and namespaces"
description = "Edit Cortex or Loki rule groups and namespaces"
keywords = ["grafana", "alerting", "guide", "group", "namespace", "cortex", "loki"]
weight = 400
+++

# Edit Cortex or Loki rule groups and namespaces

You can rename Cortex or Loki rule namespaces and groups and edit group evaluation intervals.

## Rename a namespace

> 

Reviewer: ```suggestion
A namespace contains one or more groups. To rename a namespace, find a group that belongs to the namespace, then update the namespace.

```
